webplugin/js/app/kommunicate.js

```diff
@@ -116,6 +116,48 @@ $applozic.extend(true, Kommunicate, {
             }, SET_TIMEOUT_DURATION);
         }
     },
+    updateConversationMetadata: function (userSpecificMetadata) {
```

Comment:
not able to understand why are you using 'user' ('userSpecificMetadata') in conversation related thing

Reply:
updated to conversationMetadata